test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -636,3 +636,99 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
     }
 }
+
+Describe "Bug fixes" -Tags "Feature" {
+
+    Context "InFile parameter negative tests" {
+
+        function ValidateWebError
+        {
+            param ($testCase)
+
+            It "$($testCase.Name)" {
+
+                try
+                {
+                    $sb = [scriptblock]::Create($testCase.Command)
+                    & $sb
+                    throw "CodeExecuted"
+                }
+                catch
+                {
+                    $_.FullyQualifiedErrorId | should be $testCase.ExpectedFullyQualifiedErrorId
+                }
+            }
+        }
+
+        $testCases = @(
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile ""'
+                Command = 'Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile ""'
+                ExpectedFullyQualifiedErrorId = 'WebCmdletInFileNotFilePathException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile'
+                Command = 'Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile'
+                ExpectedFullyQualifiedErrorId = 'MissingArgument,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-WebRequest -InFile c:\pathThatDoesNotExist\foo.txt'
+                Command = 'Invoke-WebRequest -Uri http://httpbin.org/post -Method Post -InFile c:\pathThatDoesNotExist\foo.txt'
+                ExpectedFullyQualifiedErrorId = 'PathNotFound,Microsoft.PowerShell.Commands.InvokeWebRequestCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-RestMethod -InFile ""'
+                Command = 'Invoke-RestMethod -Uri http://httpbin.org/post -Method Post -InFile ""'
+                ExpectedFullyQualifiedErrorId = 'WebCmdletInFileNotFilePathException,Microsoft.PowerShell.Commands.InvokeRestMethodCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-RestMethod -InFile'
+                Command = 'Invoke-RestMethod -Uri http://httpbin.org/post -Method Post -InFile'
+                ExpectedFullyQualifiedErrorId = 'MissingArgument,Microsoft.PowerShell.Commands.InvokeRestMethodCommand'
+            }
+
+            @{
+                Name = 'Validate error for Invoke-RestMethod -InFile c:\pathThatDoesNotExist\foo.txt'
+                Command = 'Invoke-RestMethod -Uri http://httpbin.org/post -Method Post -InFile c:\pathThatDoesNotExist\foo.txt'
+                ExpectedFullyQualifiedErrorId = 'PathNotFound,Microsoft.PowerShell.Commands.InvokeRestMethodCommand'
+            }
+        )
+
+        foreach ($testCase in $testCases)
+        {
+            ValidateWebError $testCase
+        }
+    }
+
+    Context "InFile parameter positive tests" {
+
+        BeforeEAch {
+
+            $filePath = Join-Path $TestDrive "test.txt"
```

Comment:
You are right. This piece of code is only to setup the test case, so I only need to create this file once.